mne/viz/utils.py

```diff
@@ -1051,39 +1063,34 @@ def f(x, y):
                     for idx in ch_idx:
                         params['ax_vscroll'].patches[idx].set_color(color)
                     break
-    else:
+    else:  # click in the background (not on a signal): set green line
         _draw_vert_line(event.xdata, params)
 
     return bads
 
 
 def _onclick_help(event, params):
     """Draw help window."""
-    import matplotlib.pyplot as plt
     text, text2 = _get_help_text(params)
 
-    width = 6
-    height = 5
+    width, height = 9, 5
 
     fig_help = figure_nobar(figsize=(width, height), dpi=80)
     fig_help.canvas.set_window_title('Help')
     params['fig_help'] = fig_help
-    ax = plt.subplot2grid((8, 5), (0, 0), colspan=5)
-    ax.set_title('Keyboard shortcuts')
-    plt.axis('off')
-    ax1 = plt.subplot2grid((8, 5), (1, 0), rowspan=7, colspan=2)
-    ax1.set_yticklabels(list())
-    plt.text(0.99, 1, text, fontname='STIXGeneral', va='top', ha='right')
-    plt.axis('off')
 
-    ax2 = plt.subplot2grid((8, 5), (1, 2), rowspan=7, colspan=3)
-    ax2.set_yticklabels(list())
-    plt.text(0, 1, text2, fontname='STIXGeneral', va='top')
-    plt.axis('off')
+    ax = fig_help.add_subplot(111)
+    celltext = [[c1, c2] for c1, c2 in zip(text.strip().split("\n"),
+                                           text2.strip().split("\n"))]
+    table = ax.table(cellText=celltext, loc="center", cellLoc="left")
+    table.auto_set_font_size(False)
+    table.set_fontsize(12)
+    ax.set_axis_off()
+    for _, cell in table.get_celld().items():
+        cell.set_edgecolor(None)  # remove cell borders
```

Comment:
Oh you already have a loop -- so doing `key, cell` then `row, col = key` and `if key == 0: cell.set_text_props(ha='right')` should help the alignment

Reply:
I've tried that, but for whatever reason this doesn't work - the text is still left-aligned...